include/client.h

```diff
@@ -11,6 +11,35 @@ namespace prism
 namespace connect
 {
 
+struct SdkVersion
+{
+    // ugly workaround for ugly bug in gcc, defining macros major() and minor()
+#undef major
+#undef minor
+
+    SdkVersion(uint8_t major, uint8_t minor, uint8_t revision)
+        : major(major)
+        , minor(minor)
+        , revision(revision)
+        , unused(0)
+    {
+    }
+
+    uint32_t toUint() const
+    {
+        return ((uint32_t)major << 24) + ((uint32_t)minor << 16) + ((uint32_t)revision << 8);
+    }
+
+    std::string toString() const;
+
+    uint8_t major;
+    uint8_t minor;
+    uint8_t revision;
```

Comment:
major is likely be small number as 1, 2 or 3 (even firefox/google chrome versions are below 100)
minor should be enough
revision starts with 0 for each fixed major and minor to identify changes, that doesn't change API e.g. fixes, refactoring.